test/e2e/node_feature_discovery_test.go

```diff
@@ -1044,3 +1122,20 @@ func getNode(nodes []corev1.Node, nodeName string) corev1.Node {
 	}
 	return corev1.Node{}
 }
+
+// Function to compare two string slices for equality regardless of order
+func areEqualLists(list1, list2 []string) bool {
+	// Convert lists to sets
+	set1 := make(map[string]struct{})
+	for _, elem := range list1 {
+		set1[elem] = struct{}{}
+	}
+
+	set2 := make(map[string]struct{})
+	for _, elem := range list2 {
+		set2[elem] = struct{}{}
+	}
+
+	// Check if sets are equal
+	return reflect.DeepEqual(set1, set2)
```

Comment:
You could use the sets package from apimachinery (`k8s.io/apimachinery/pkg/util/sets`)

```suggestion
	set1 := sets.NewString(list1...)
	set2 := sets.NewString(list2...)
	return set1.Equal(set2)
```

Reply:
done